docs/current_docs/integrations/depot.mdx

```diff
@@ -0,0 +1,65 @@
+---
+slug: /integrations/depot
+---
+
+# Depot
+
+Depot provides managed, Dagger-powered GitHub Actions runners. These runners, which serve as drop-in replacements for GitHub's own runners, are optimized to make it faster and easier to run Dagger pipelines in GitHub repositories.
+
+## How it works
+
+When running a CI pipeline on a Dagger-powered Depot GitHub Runner, the general workflow looks like this:
+
+1. GitHub emits a repository event which triggers a workflow
+1. Depot receives this event, process it, and runs one or more jobs in the workflow, each running in an isolated Depot GitHub Runner.
+1. Each Depot GitHub Runner is already connected to an already running Dagger Engine (this exists outside of the Depot GitHub Runner). Previous runs may have cached data in this external Dagger Engine.
+1. The "Dagger for GitHub" GitHub Action installs the Dagger CLI, and then calls the specified sub-command, module, function name, and arguments.
```

Comment:
The Dagger CLI is pre-installed into our runner.

Reply:
Fixed